browser/help/topics/structural-variants/sv-effect-overview.md

```diff
@@ -13,16 +13,20 @@ The functional consequence per SV was determined in the order below:
 
 ### Main functional consequences
 
-  1. ![](https://placehold.it/15/D43925/000000?text=+) **Predicted loss-of-function (pLoF)**: SV is predicted to delete the gene or truncate the gene product.
-  2. ![](https://placehold.it/15/7459B2/000000?text=+) **Intragenic exonic duplication (IED)**: SV is predicted to result in duplicated exons within the gene, without extending beyond the boundaries of the open reading frame.
-  3. ![](https://placehold.it/15/2376B2/000000?text=+) **Copy gain (CG)**: SV is predicted to result in an entire additional intact copy of the gene.
+1. ![](https://placehold.it/15/D43925/000000?text=+) **Predicted loss-of-function (pLoF)**: SV is predicted to delete the gene or truncate the gene product.
+2. ![](https://placehold.it/15/7459B2/000000?text=+) **Intragenic exonic duplication (IED)**: SV is predicted to result in duplicated exons within the gene, without extending beyond the boundaries of the open reading frame. (New in gnomAD v3)
```

Comment:
Are these icons placeholders so you can immediately see they need review? This comment is pretty much just personal curiosity.

Reply:
TBH I'm not sure of why these icons are here or why it makes sense to use a random web service to render them, but that's how I found it in the original.